src/renderers/shaders/ShaderLib/skybox_vert.glsl.js

```diff
@@ -0,0 +1,26 @@
+export default /* glsl */`
+varying vec3 vWorldPosition;
+
+#include <common>
+
+#if NUM_CLIPPING_PLANES == 0
+	#if ! defined( VARYING_VVIEWPOSITION )
+   		varying vec3 vViewPosition;
```

Comment:
Can you please explain this? Your code is throwing console warnings that `vViewPosition` is not used by the fragment shader.

Reply:
Removed vViewPosition